drv/lpc55-gpio/src/main.rs

```diff
@@ -137,8 +145,143 @@ impl idl::InOrderPinsImpl for ServerImpl<'_> {
             core::ptr::write_volatile(base as *mut u32, conf);
         }
 
+        // If the GPIO pin is configured for interrupts, then
+        // interrupt configuration is done before pin configuration.
+        // INPUTMUX from UM11126:
+        // Once set up, no clocks are required for the input multiplexer to
+        // function. The system clock is needed only to write to or read from
+        // the INPUT MUX registers. Once the input multiplexer is configured,
+        // disable the clock to the INPUT MUX block in the AHBCLKCTRL register.
+        // See: Section 4.5.17 “AHB clock control 0”.
+        match pint_slot {
+            PintSlot::None => {}
```

Comment:
This basically feels like 
```
match pint_slot {
Some(slot) => ...
_ => {}
}

Reply:
Cleaned this up using Hubpack which then simplifies everything else.